src/lib/pick-props.js

```diff
@@ -0,0 +1,13 @@
+const isPlainObject = obj =>
+  typeof obj === 'object' && obj !== null && obj.constructor === Object && Object.prototype.toString.call(obj) === '[object Object]'
+
```

Comment:
Didn't see your test, so I ported over a bunch of the tests from lodash.  There were some areas where they weren't quite the same (such as if the object contained a custom defined "constructor" property), so it's probably a good thing in the end.